src/components/header/_header.scss

```diff
@@ -0,0 +1,110 @@
+@import "../../_base.scss";
+
+.header {
```

Comment:
Where is this div with header class

Reply:
@faran4engg  its a Wrapper in Header component